server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -92,7 +95,20 @@ export function create(
     router.post("/repos/:ignored?/:tenantId/git/summaries",
         throttle(throttler, winston, commonThrottleOptions),
         (request, response, next) => {
-            const summaryP = createSummary(request.params.tenantId, request.get("Authorization"), request.body);
+            // request.query type is { [string]: string } but it's actually { [string]: any }
+            // Account for possibilities of undefined, boolean, or string types. A number will be false.
+            const initial: boolean | undefined = typeof request.query.initial === "undefined"
+                ? undefined
+                : typeof request.query.initial === "boolean"
+                    ? request.query.initial
+                    : request.query.initial === "true";
+            Lumberjack.info("Writing summary", { initial, param: request.params.initial, params: request.params });
```

Comment:
Can we extract documentId and tenantId at this point and give them as parameters to Lumberjack? That would make it easier for querying later

Reply:
Ah! This was for debugging and I forgot to remove it! Thanks for the catch!